test/unit/chips/ChipSet.test.tsx

```diff
@@ -414,6 +414,31 @@ test('chip is rendered with computeBoundingRect method prop if is not filter var
   assert.equal(chip.props.computeBoundingRect, null);
 });
 
+test('basic variant ChipSet will not throw error if chip missing id', () => {
+  const stub = <ChipSet><Chip /></ChipSet>;
+  const render = () => shallow<ChipSet>(stub);
+  assert.isOk(render);
```

Comment:
what about doesnotthrow. I think that is a method

Reply:
good catch.